data/templates/esx-ks

```diff
@@ -171,6 +183,9 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
       esxcli network ip interface remove -i <%=vmkname%>
       esxcli network ip interface add -i <%=vmkname%> -p <%=n.device%>
       esxcli network ip interface ipv4 set -i <%=vmkname%> -t dhcp
+      sleep 60
```

Comment:
To allow the interface to dhcp and gets an IP address before we can query that IP to post lookup.